atecc608a/mbed_app.json

```diff
@@ -1,18 +1,15 @@
 {
     "target_overrides": {
         "*": {
-            "target.features_add" : ["EXPERIMENTAL_API"],
+            "target.features_add" : ["EXPERIMENTAL_API", "PSA"],
+            "extra_labels_add": ["MBED_PSA_SRV"],
```

Comment:
`[ERROR] Attempt to override undefined parameter 'app.extra_labels_add' in 'application[*]'
[mbed] Working path "/builds/workspace/mbed-os-ci_build-example-GCC_ARM/examples/mbed-os-example-atecc608a/atecc608a" (program)`. This should be `target.extra_labels_add`

This is from Mbed OS master testing (see the last PR log https://github.com/ARMmbed/mbed-os/pull/14661#issuecomment-842148502).

Reply:
I'm working on this. Somehow it works with Mbed CLI 2.